helm-charts/sep-service/values.yaml

```diff
@@ -0,0 +1,134 @@
+fullName: anchor-platform
+namespace: sandbox
+service:
+   containerPort: 8080
+   servicePort: 8080
+   replicas: 1
+   type: NodePort
+   name: sep
+image:
+   repo: stellar
+   name: anchor-platform
+   tag: 022afab
+   pullPolicy: Always
+ingress:
+   labels:
+     app: appy
+     test: label_test
+   annotations:
+     kubernetes.io/ingress.class: "public"
+     cert-manager.io/cluster-issuer: "default"
+     ingress.kubernetes.io/ssl-redirect: "true"
+     ingress.kubernetes.io/browser-xss-filter: "true"
+     ingress.kubernetes.io/frame-deny: "true"
+     ingress.kubernetes.io/content-type-nosniff: "true"
+     ingress.kubernetes.io/hsts-max-age: "31536000"
+     ingress.kubernetes.io/hsts-include-subdomains: "true"
+   tls:
+     host: "www.stellaranchordemo.com"
+     secretName: anchor-platform-sep-server-cert
+   rules:
+      hosts:
+        - host: "{{ .Values.ingress.tls.host }}"
+          path: /
+          pathType: Prefix
+          backend:
+            servicePort: "{{ .Values.service.containerPort }}"
+            serviceName: "{{ .Values.fullName }}-svc-{{ .Values.service.name }}"
+stellar:
+   anchor:
+      config: in-memory
+      app-config:
+         type: config-spring-property                # Activate [config-spring-property] module
+         settings: "app-config"                        # The location of the configuration data
+      data-access:
+         type: data-spring-jdbc                      # Activate [config-spring-jdbc] module.
+         settings: data-spring-jdbc-sqlite           # The location of the configuration data in this file.
+      logging:
+         type: logging-logback
+         settings: logging-logback-settings
+   app_config:
+      app:
```

Comment:
Structure here is quite complex and multi-level. Maybe it would make sense to remove the `stellar` level completely to reduce amount of indentation needed? We could go even further and just have things like `sep1` and `sep12` at the top level. WDYT?

Also I think some names might be confusing to some users. For example we have `stellar.anchor.app-config` but also `stellar.app_config`. Maybe it's possible to rename one of them to clarify what they mean?

Reply:
I think I will do this but will delay this refactor after initial merge if thats ok and once we have it into our test pipeline.